.github/workflows/android-publish.yml

```diff
@@ -95,7 +111,7 @@ jobs:
       - name: Create build and upload to play console
         if: ${{ inputs.BETA_RELEASE != 'true' }}
         run: |
-          cd android/scripts
+          cd ${{ inputs.ANDROID_SERVICE_LOCATION }}/scripts
```

Comment:
Please remove /scripts as well..

As the ANDRIOD service location will contain the complete location